File: frontend/lib/models/statistic_table_column.dart
class StatisticTableColumn {
  final String? key;
  final String? label;
  final String? unit;
  final int? order;

  StatisticTableColumn({this.key, this.label, this.unit, this.order});

  factory StatisticTableColumn.fromJson(Map<String, dynamic> json) {
    return StatisticTableColumn(
      key: json['key_name'] as String?,
      label: json['label'] as String?,
      unit: json['unit'] as String?,
      order: json['order'] as int?
    );
  }
}

File: backend/app/Models/StatisticTableColumn.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class StatisticTableColumn extends Model
{
    protected $table = 'statistic_table_columns';

    protected $fillable = [
        'table_id',
        'key_name',
        'label',
        'unit',
        'order_index',
    ];

    public function table()
    {
        return $this->belongsTo(
            StatisticTable::class,
            'table_id'
        );
    }
}
